CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# How to contribute
+
+There are lots of ways to contribute to the Uno Platform and we appreciate the help from the community. You can provide feedback, report bugs, give suggestions, contribute code, and participate in the platform discussions.
```

Comment:
There are lots of ways to contribute to the _Source Generation Tasks_

Reply:
I will update it thanks